APMrover2/GCS_Mavlink.cpp

```diff
@@ -55,9 +55,9 @@ void Rover::send_heartbeat(mavlink_channel_t chan)
     base_mode |= MAV_MODE_FLAG_CUSTOM_MODE_ENABLED;
 
     gcs().chan(chan-MAVLINK_COMM_0).send_heartbeat(MAV_TYPE_GROUND_ROVER,
-                                            base_mode,
```

Comment:
Making these whitespace changes in the same commit as adding the packetreceived hook isn't a great idea.

Separate commits at the very least.  Preferably not at all, as it creates a point in the file history where `git blame` becomes useless.  Your whitespace-only commit becomes a point where finding where a logic error was introduced becomes problematic.